vdirsyncer/storage/google.py

```diff
@@ -94,6 +94,11 @@ def _save_token(token):
 class GoogleCalendarStorage(dav.CaldavStorage):
     __doc__ = '''Google calendar.
 
+    .. note:: You have to poperly set up ``collections``, otherwise errors like
```

Comment:
The example in "tutorial" (for ownClould) has `collections = null` and "config" does not list that as a required field. Thus I thought one only has to set `collections` when syncing non-default adressbooks/calendars.